hazelcast/src/test/java/com/hazelcast/map/LocalIndexStatsTest.java

```diff
@@ -635,6 +635,7 @@ public void testIndexStatsAfterMapDestroy() {
         assertTrue(valueStats().getTotalUpdateLatency() > 0);
         assertTrue(valueStats().getAverageHitLatency() > 0);
 
+        ensureSafeState();
```

Comment:
Can you explain how this helps with this test failure?

Reply:
destroy is not a partition operation, to be sure there is no race condition with any other partition operation, i have put that safe-state check before destroy. It is useful to eliminate one more possible case when looking into next failure.

Reply:
OK, let's try this